src/Common/ProcessImporter.ts

```diff
@@ -313,15 +315,17 @@ export class ProcessImporter {
 
                     if (!group.inherited) {
                         //create the group if it's not inherited
-                        const createGroup = Utility.toCreateGroup(group);
+                        const createGroup = Utility.toCreateGroup(group, groupIndex);
                         newGroup = await this._createGroup(createGroup, page, section, witLayout, payload);
                         group.id = newGroup.id;
                     }
 
-                    for (const control of group.controls) {
+                    for (let controlIndex = 0; controlIndex < group.controls.length; controlIndex++) {
+                        let control = group.controls[controlIndex];
```

Comment:
nit - use `const` than `let` ?

Reply:
Are you referring to changing to `const control = groups.controls[controlIndex];`? It looks like the compiler won't allow changing `const controlIndex = 0; controlIndex < group.controls.length; controlIndex++` because it can't be incremented as constant.